app/src/main/java/com/sopt/now/MainActivity.kt

```diff
@@ -1,15 +1,28 @@
 package com.sopt.now
 
 import android.os.Bundle
-import androidx.activity.enableEdgeToEdge
 import androidx.appcompat.app.AppCompatActivity
-import androidx.core.view.ViewCompat
-import androidx.core.view.WindowInsetsCompat
+import com.sopt.now.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityMainBinding
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        // SignUpActivity로부터 전달된 값들을 받아옴
+        val userId = intent.getStringExtra("userId")
+        val userPw = intent.getStringExtra("userPw")
+        val userName = intent.getStringExtra("userName")
+        val userMbti = intent.getStringExtra("userMbti")
 
+        // 받아온 값을 각각의 TextView에 설정
+        binding.tvUserId.text = "ID: $userId"
+        binding.tvUserPw.text = "Password: $userPw"
+        binding.tvUserName.text = "Name: $userName"
+        binding.tvUserMbti.text = "MBTI: $userMbti"
```

Comment:
위에 말씀드렸던 Scope 함수중 `with`라는 친구의 활용 방법입니다!

이렇게 사용하면 불필요한 반복도 줄여주고, with(binding) 내부에 있는 코드들이기 때문에 "저 부분은 UI를 조정하는 부분이구나" 라는 느낌을 줘서 다른 사람이 읽기에도 편해집니다 :)

```suggestion
   with(binding) {
        tvUserId.text = "ID: $userId"
        tvUserPw.text = "Password: $userPw"
        tvUserName.text = "Name: $userName"
        tvUserMbti.text = "MBTI: $userMbti"
    }
```

Reply:
신세계네요. 감사합니다!!